app/models/flat_reasons_for_rejection_extract.rb

```diff
@@ -0,0 +1,183 @@
+class FlatReasonsForRejectionExtract
+  include ActiveModel::Model
+
+  def initialize(structured_rejection_reasons)
+    @structured_rejection_reasons = structured_rejection_reasons
+  end
+
+  def candidate_behaviour?
```

Comment:
Not sure whether this type of method returns a `'Yes'` or `'No'` string or a bool. It kind of breaks convention if its a string.